fixtures/pipelighting/pipedimmer.json

```
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "PipeDimmer",
+  "categories": ["Dimmer"],
+  "meta": {
+    "authors": ["CrewArt"],
+    "createDate": "2024-10-23",
+    "lastModifyDate": "2024-10-23"
+  },
+  "links": {
+    "manual": [
+      "https://static1.squarespace.com/static/60cdcfbce889114bd384468f/t/6579d10162771a6b7f558d50/1702482184431/PIPE+USER+MANUAL+PW_42+V1.pdf"
+    ],
+    "productPage": [
+      "https://www.pipelighting.com/"
+    ],
+    "video": [
+      "https://www.pipelighting.com/videos"
```

```suggestion
      "https://www.youtube.com/watch?v=EZrut0S0vw0"
```

This one is for the PipeWide 84, but it looks like the most relevant video here.